content/guides/overview/choosing-e2e-or-ct.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'Choosing a Testing Type: End-to-end or Component Tests?'
+---
+
+<Alert type="info">
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- What is end-to-end testing and component testing
+- Considerations for each testing type
+- How to choose which test type based on your scenario
+
+</Alert>
+
+One of the first decisions you will need to make on your testing journey is what
+type of test to create. Cypress offers two options: end-to-end (e2e) and
+component tests. There are benefits and considerations for each choice, and the
+decision will depend on the needs of what you are currently trying to
+accomplish. In the end, you will probably have a combination of both types of
+tests for your app, but how do you choose right now?
+
+Let's go over each of these test types, the benefits they bring, things to
+consider, and scenarios for each.
+
+## What is End-to-end Testing?
+
+End-to-end testing is a technique that tests your app from the web browser
+through to the back end of your application, as well as testing integrations
+with third-party APIs and services. These types of tests are great at making
+sure your entire app is functioning as a cohesive whole.
+
+Cypress runs e2e tests the same way users interact with your app by using a real
+browser, visiting URLs, viewing content, clicking on links and buttons, etc.
+Testing this way helps ensure your tests and the user's experience are the same.
+
+Writing end-to-end tests in Cypress can be done by developers building the
+application, specialized testing engineers, or a quality assurance team
+responsible for verifying an app is ready for release. Tests are written in code
+with an API that simulates the steps a that a real user would take.
+
+End-to-end tests are great at verifying your app runs as intended, from the
+front end to the back end. However, end-to-end tests can be more difficult to
+set up, run, and maintain. There are often infrastructure needs in setting up a
+backend for testing purposes. Your team will need to develop a
+[strategy](https://docs.cypress.io/guides/getting-started/testing-your-app#Testing-strategies)
+on how to handle this complexity.
+
+<Alert type="success">
+
+### Benefits of end-to-end tests:
```

Comment:
I think we should lose the `:`: in all headers, because they look a bit weird in the right sidebar